test/e2e/flowaggregator_test.go

```diff
@@ -1815,6 +1837,37 @@ func createToExternalTestServer(t *testing.T, data *TestData) *PodIPs {
 	return serverIPs
 }
 
+func getAndCheckFlowAggregatorMetrics(data *TestData) error {
+	errLogs := []string{}
```

Comment:
I don't think you need this: if it's a "fatal" error, interrupt the polling, if it's a possibly transient error, log it and continue

Reply:
I've updated this func. Only use fatal when we cannot Unmarshal the output, otherwise, we print the log instead. Thanks.